console/console-init/ui/src/schema/iot_device.ts

```diff
@@ -34,3 +34,9 @@ export interface IIoTDevicesResponse {
     }>;
   };
 }
+
+export interface IIoTDevicesCountResponse {
```

Comment:
don't need to create separate interface for each schema, we have already use that one.